Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:alpine
```

Comment:
the next step will be introducing multistage builds, so there will be no need in using alpine